### PR TITLE
Version/3.10

### PR DESCRIPTION
**Pull request by lbudai**
Don't merge, just review.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: lib/pragma-grammar.ym [hunk 1]
```diff
@@ -60,10 +60,10 @@ __process_version_string(gchar *value)
...
   if (p)
     {
-      major = strtol(value, &end, 10);
+      major = strtol(value, &end, 16);
       if (end == p)
         {
-          minor = strtol(p+1, &end, 10);
+          minor = strtol(p+1, &end, 16);
           if (end)
             {
```
Comment (bazsi) [inline lib/pragma-grammar.ym]: Hmm I disagree with this, why would we want to parse them in hex?

File: lib/versioning.h [hunk 2]
```diff
@@ -112,11 +113,12 @@
 #define VERSION_VALUE_3_7  0x0307
 #define VERSION_VALUE_3_8  0x0308
 #define VERSION_VALUE_3_9  0x0309
+#define VERSION_VALUE_3_10 0x0310
```
Comment (bazsi) [inline lib/versioning.h]: Ah, this is the reason, 0x030A ?

Comment (lbudai) [inline lib/versioning.h]: exactly.

Comment (bazsi) [inline lib/versioning.h]: I mean why couldn't we use 0x030a?

Comment (lbudai) [inline lib/versioning.h]: We could use, of course...

Comment (lbudai) [inline lib/versioning.h]: done

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
@bazsi : It is ready to merge (I've added the updated NEWS.md file ) .

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**